docker-compose.yaml

```diff
@@ -281,7 +245,7 @@ services:
 
   flower:
     <<: *airflow-common
-    command: ["celery", "flower"]
+    command: celery flower
```

Comment:
This would be a welcome PR back into `apache/airflow` I bet!

Reply:
this change is actually reverting to airflow's original command: i made the change while trying to get flower to work 🙈